src/sherpa_ai/utils.py

```diff
@@ -111,10 +111,21 @@ def count_string_tokens(string: str, model_name: str) -> int:
     return len(encoding.encode(string))
 
 
-def chunk_and_summarize(text_data: str,  question: str, open_ai_key: str, link: str, team_id: str = None, user_id: str = None):
+def chunk_and_summarize(
+    text_data: str,
+    question: str,
+    open_ai_key: str,
+    link: str,
+    team_id: str = None,
+    user_id: str = None,
+):
     llm = SherpaOpenAI(
-        temperature=cfg.TEMPRATURE, openai_api_key=open_ai_key, user_id=user_id, team_id=team_id)
-   
+        temperature=cfg.TEMPRATURE,
```

Comment:
the config is not imported because of this it fails when you try to run a prompt with a link

Reply:
I added the missing import